src/screen.cpp

```diff
@@ -72,7 +72,7 @@ static float get_pixel_ratio(GLFWwindow *window) {
 }
 Screen::Screen()
     : Widget(nullptr), mGLFWWindow(nullptr), mNVGContext(nullptr),
-      mCursor(Cursor::Arrow), mBackground(0.3f, 0.3f, 0.32f),
+      mCursor(Cursor::Arrow), mBackground(0.3f, 0.3f, 0.32f), mCaption(""),
```

Comment:
Ahhh you are right, I was being over zealous, since I thought some un-initialized params in the constructor were causing issues. I will remove this.